.projenrc.js

```diff
@@ -8,6 +8,9 @@ const project = new GemeenteNijmegenCdkApp({
   deps: ['@gemeentenijmegen/aws-constructs',
     '@gemeentenijmegen/utils',
     '@aws-sdk/client-s3',
+    '@aws-sdk/client-apigatewayv2',
+    '@aws-sdk/lib-dynamodb',
+    '@aws-sdk/client-dynamodb',
     'https',
     'aws-sdk',
```

Comment:
Deze kan nu vgm weg (alles maakt nu gebruik vd v3 sdk.

Reply:
Correct, dep is verwijderd